<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T.N.M.I</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_carddraw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>카드 뽑기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gyoran">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>교란순열</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_coupon">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>쿠폰 콜렉터</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_needle">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>200</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>뷔퐁의 바늘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ball">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>200</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>공 뽑기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>-10</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>T.N.M.I</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>70</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Transcendental Number Measuring Instrument</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
